cli/CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ _Released 12/3/2024 (PENDING)_
 - Cypress Component Testing no longer supports `Vue` version 2. Addresses [#30295](https://github.com/cypress-io/cypress/issues/30295).
 - Cypress Component Testing no longer supports `Next.js` versions 10, 11, 12, and 13. Addresses [#29583](https://github.com/cypress-io/cypress/issues/29583).
 - Cypress Component Testing no longer supports `Angular` versions 13, 14, 15, and 16. The minimum supported version is now `17.2.0` in order to fully support Angular [signals](https://angular.dev/guide/signals). Addresses [#29582](https://github.com/cypress-io/cypress/issues/29582). Addressed in [#30539](https://github.com/cypress-io/cypress/pull/30539).
+- Cypress Component Testing no longer support `Svelte` version 3. Addresses [#30492](https://github.com/cypress-io/cypress/issues/30492)
```

Comment:
```suggestion
- Cypress Component Testing no longer supports `Svelte` version 3. Addresses [#30492](https://github.com/cypress-io/cypress/issues/30492)
```

Reply:
I'd add a period to the end as well

Reply:
updated [60dd249](https://github.com/cypress-io/cypress/pull/30572/commits/60dd24993271dac64fbffc95702152a10adfb54b)